cpp_exercise/src/main/ipcreceiver.cc

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include "src/lib/linuxipclib.h"
+
+
+
+int main(int argc,char *argv[]) {
+
+    std::vector<std::string> commandline;
+   try{
+
+    CommandOption cmo("ipcreceiver",argc,argv);
+    commandline = cmo.getCommand();
+
+   } catch(IPCException & e){
+       std::cout<<e.what()<<std::endl;
+       exit(EXIT_FAILURE); 
+   }
+    try{
```

Comment:
If you catch the same type of exception I don't see reason to put code into separate blocks

Reply:
Fixed